portality/lib/anon.py

```diff
@@ -0,0 +1,14 @@
+import hashlib
+from portality.core import app
+
+
+def anon_email(email):
+    return hashlib.sha256(app.config['ANON_SALT'] + email).hexdigest() + '@example.com'
+
+
+def basic_hash(id_):
+    return hashlib.sha256(app.config['ANON_SALT'] + id_).hexdigest()
+
+
+def anon_name(name):
+    return 'Stacy ' + basic_hash(name)
```

Comment:
Is there a reason for using the name "Stacy" here.  I get that this might make the names more readable up-front, but having them all start with the same name might be confusing.  I'd either just do away with this, or we could randomly generate names from a set of first-name last-name pairs, which is a trick I've used before to make realistic looking data.

Reply:
Just a way to distinguish the data is a name at all. Replaced with human-like names quickly using Faker, a neat library we could use in lots of places.